.github/workflows/unit.yaml

```diff
@@ -0,0 +1,26 @@
+name: Unit tests
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
Should this be:

```suggestion
      - main
```

Reply:
Ups, it seems that it has to be changed everywhere!! Good catch @richardcase